### PR TITLE
feat(deps): support latest Cerebral 0.33 and new releases

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

**Comment by Guria** [comment]
why not to use alias instead of factory?

**Comment by Guria** [comment]
https://github.com/cerebral/cerebral-module-recorder/blob/newContextSignals/index.js#L33
replace `isSync` with `immediate` option

**Comment by Guria** [comment]
set [`cerebral@0.33.31`](https://github.com/cerebral/cerebral/releases/tag/v0.33.31) as minimal peerVersion

**Comment by Guria** [comment]
port [this changes](https://github.com/cerebral/cerebral-module-recorder/pull/5/files#diff-168726dbe96b3ce427e7fedce31bb0bcR153) too

**Comment by christianalfoni** [comment]
1. Do not have access to the alias in 0.34 inside actions (without adding a context provider for modules)
2. Changed to immediate now :)
3. Changed minimal version
4. Moved over the IF thingy there as well. Tests and deps was up to date

File: index.js [hunk 1]
```diff
@@ -166,6 +169,15 @@ module.exports = function (options) {
 
     module.alias(MODULE)
 
+    if (module.addContextProvider) {
+      controller.addContextProvider({
```
Comment (Guria) [inline index.js]: we already have MODULE const. How about:
```js
var context = {}
context[MODULE] = { path: module.path }
controller.addContextProvider(context)
```

File: index.js [hunk 2]
```diff
@@ -166,6 +169,15 @@ module.exports = function (options) {
 
     module.alias(MODULE)
 
+    if (module.addContextProvider) {
```
Comment (Guria) [inline index.js]: maybe better to check `controller.addContextProvider` here?

File: signals/index.js [hunk 3]
```diff
@@ -1,43 +1,63 @@
-function play (arg) {
-  arg.module.services.seek(0)
-  arg.module.state.merge([], {
+function play (context) {
+  var modulePath = context.modules ? context.modules['cerebral-module-recorder'].path : context['cerebral-module-recorder'].path
+  var services = modulePath.reduce(function (services, key) {
```
Comment (Guria) [inline signals/index.js]: maybe `var services = getModuleServices(context)`?

File: index.js (hunk 1 above)
Comment (christianalfoni) [inline index.js]: agreed

File: index.js (hunk 2 above)
Comment (christianalfoni) [inline index.js]: agreed

File: signals/index.js (hunk 3 above)
Comment (christianalfoni) [inline signals/index.js]: agreed :)

**Comment by christianalfoni** [comment]
Okay, implemented all your suggestions, very good :)

**Comment by Guria** [comment]
LGTM

**Comment by Guria** [comment]
please squash and use:
```js
"peerDependencies": {
  "cerebral": "^0.33.31 || ^0.34.0-rc.4"
},
```

**Pull request merged**